rllib/algorithms/algorithm_config.py

```diff
@@ -2470,6 +2499,8 @@ def offline_data(
             self.input_read_method_kwargs = input_read_method_kwargs
         if input_read_schema is not NotProvided:
             self.input_read_schema = input_read_schema
+        if prelearner_class is not NotProvided:
+            self.prelearner_class = prelearner_class
```

Comment:
Wait, where do these get assigned?
`input_read_method_kwargs`
`map_batches_kwargs`

Reply:
Up in the file. Where all attributes get default values.

Reply:
No, sure, but if the user sets them through the `offline_data()` method, currently nothing would happen, correct?

Reply:
Ah, sorry, didn't see this. I think b/c it hadn't been changed in this PR. All good.